tests/experimental/liveness/liveness.slang.expected

```diff
@@ -0,0 +1,211 @@
+result code = 0
```

Comment:
I get very concerned about tests where the expected output is the exact textual output of the compiler, since it makes it very easy for small changes to the compiler implementation to break all such tests and require slow manual checking.

For this specific case, I can't see an easy way around the problem, but once we have some path where we are getting end-to-end codegen with live ranges (e.g., by emitting the relevant SPIR-V opcodes, even if they aren't legal for Vulkan at present), then we can set this up like our cross-compile tests where we have both a Slang input and a GLSL reference, but the comparison is on the SPIR-V emitted for both.

Reply:
Could use SPIR-V with the later change?